client/src/rpc_client.rs

```diff
@@ -703,103 +699,6 @@ impl RpcClient {
         Ok(hash)
     }
 
-    pub fn get_token_account(&self, pubkey: &Pubkey) -> ClientResult<Option<UiTokenAccount>> {
```

Comment:
Nuke things I added in for spl-token-cli but no longer need because the decimals is returned from RPC directly now